### ISSUE TITLE
Add in progression to the raml

### ISSUE THREAD

**Issue by jonyeezs** [cvent/node-octopus-deploy-client#1]
Octopus api has a resource to get the release progression of a project: https://github.com/OctopusDeploy/OctopusDeploy-Api/wiki/Progressions

Could this client support that?

I'm not sure what version of the api has this call.

**Comment by brentm5** [comment]
Yes we accept PRs!  All you would have to do is update the RAML to add the endpoint.  If everything looks good I can merge and publish another version of this.
